src/App/Markdown.php

```diff
@@ -21,4 +21,27 @@
 
 class Markdown
 {
+    public function __construct($argument1)
```

Comment:
add `Writer` type hint berdasarkan : `$this->beConstructedWith($writer);` di specnya